src/main/java/seedu/edrecord/model/person/Person.java

```diff
@@ -100,7 +100,7 @@ public boolean equals(Object other) {
                 && otherPerson.getPhone().equals(getPhone())
                 && otherPerson.getEmail().equals(getEmail())
                 && otherPerson.getInfo().equals(getInfo())
-                && otherPerson.getModule().equals(getModule())
+                && otherPerson.getModule().isSameModule(getModule())
```

Comment:
What's the rationale behind using the looser def for module?

Reply:
Since Assignments are stored under Module, `Module#equals()` now has to check for equality using both module code and the assignment list. Hence some tests which uses module equality need to be changed to `Module#isSameModule()` which is equality based on module code only.